### ISSUE TITLE
fix incorrect alphabetization in Author index

### ISSUE THREAD

**Issue by juliaflanders** [Digital-Humanities-Quarterly/dhq-journal#73]
The entry for:
```
Čejka, Martin, Czech University of Life Sciences Prague
[Cuneiform Stroke Recognition and Vectorization in 2D Images](http://www.digitalhumanities.org/dhq/vol/18/1/000733/000733.html), 2024: v18 n1, [co-authored]

```
is being alphabetized as "ejka" (the initial Č is being ignored)

**Comment by amclark42** [comment]
I am fixing this as part of my push to make links relative within DHQ.

**Comment by amclark42** [comment]
Fixed in #89. The solution will be part of the static site rollout.
